ReleaseTooling/Sources/ZipBuilder/main.swift

```diff
@@ -236,11 +236,8 @@ struct ZipBuilderTool: ParsableCommand {
                                            logsOutputDir: outputDir?
                                              .appendingPathComponent("build_logs"))
 
-    // Populate the platforms list if it's empty. This isn't a great spot, but the argument parser
```

Comment:
Was this fixed in 0.3 of the Argument Parser?

Reply:
Yes this is fixed in 0.3. Also, default setup for parameters are  updated as well.
Instead of using `default: "default_value`, a default value could be assign to an option. So not just array, string/int could be assigned directly.